buildbot/master/files/config/factories.py

```diff
@@ -211,10 +213,11 @@ def make_step(self, command):
                 step_kwargs['logfiles'][logfile] = logfile
 
             # Provide environment variables for s3cmd
-            elif arg == './etc/ci/upload_nightly.sh':
+            elif arg == './etc/ci/upload_nightly.sh' or
+                 next(args) == r'.\etc\ci\upload_nightly.sh':
```

Comment:
Wait, that's definitely not a Python script but bash - will bash scripts run properly on MSVC?